### PR TITLE
add the map to the upload form

### PR DESCRIPTION
**Pull request by tiagorbf**

### CODE REVIEW

File: website-hugo/src/themes/covid19/assets/js/covid.js [hunk 1]
```diff
@@ -4,6 +4,55 @@ var startDate = new Date();
 var endDate = new Date();
 var formState = {};
 
+var mymap = L.map('mapid').setView([38.7436214, -9.1953085], 13);
```
Comment (woolfg) [inline website-hugo/src/themes/covid19/assets/js/covid.js:7]: would suggest around 47,11 and level 5 to cover europe

File: website-hugo/src/themes/covid19/assets/js/covid.js [hunk 2]
```diff
@@ -4,6 +4,55 @@ var startDate = new Date();
...
+
+function removeMarker(e){
+    mymap.removeLayer(markers[e]);
+    delete markers[parseInt(e)];
+}
+
+function getMapLocations(){
+    var locations = [];
+    for (var key of Object.keys(markers)) {
+        coordinates = {
```
Comment (woolfg) [inline website-hugo/src/themes/covid19/assets/js/covid.js:46]: google format is:
```{"locations":[{"timestampMs":"1583191948838","latitudeE7":512458939,"longitudeE7":67860124,"accuracy":15,"altitude":88,"verticalAccuracy":3},{"timestampMs":"1583193338620","latitudeE7":512458939,"longitudeE7":67860124,"accuracy":15,"altitude":88,"verticalAccuracy":3},```

the important part is "timestampMs","latitudeE7","longitudeE7" - timestamp has to be unique, so maybe start with the date 1 day ago and then just ++ with every location?

Comment (woolfg) [reply website-hugo/src/themes/covid19/assets/js/covid.js:46]: attention: long, lat is an int.

File: website-hugo/src/themes/covid19/layouts/shortcodes/uploadform.html [hunk 3]
```diff
@@ -1,106 +1,113 @@
...
-        <div class="questionblock hidden" id="response">
-            <label>
-                <div id="uploadstatus" class="informationbox"></div>
-            </label>
-
-            <div class="question">
-                <div id="uploadinformation" class="informationbox"></div>
-            </div>
+        <label>
+            Manually select the places you visited in the past 15 days
```
Comment (woolfg) [inline website-hugo/src/themes/covid19/layouts/shortcodes/uploadform.html:88]: i18? I would add `(just klick on the map to add places)`

File: website-hugo/src/themes/covid19/assets/js/covid.js (hunk 1 above)
Comment (tiagorbf) [reply website-hugo/src/themes/covid19/assets/js/covid.js:7]: I was even thinking on having different coordinates depending on the country. But yes, for now makes sense

File: website-hugo/src/themes/covid19/layouts/shortcodes/uploadform.html (hunk 3 above)
Comment (tiagorbf) [reply website-hugo/src/themes/covid19/layouts/shortcodes/uploadform.html:88]: yup. will do

File: website-hugo/src/themes/covid19/assets/js/covid.js (hunk 2 above)
Comment (woolfg) [reply website-hugo/src/themes/covid19/assets/js/covid.js:47]: lat, long is not an int

**Pull request merged**